sync/class.jetpack-sync-functions.php

```diff
@@ -203,26 +210,26 @@ public static function get_protocol_normalized_url( $callable, $new_value ) {
 		$option_key = self::HTTPS_CHECK_OPTION_PREFIX . $callable;
 
 		$parsed_url = wp_parse_url( $new_value );
-		if ( ! $parsed_url ) {
+		if ( empty ( $parsed_url['scheme'] ) ) {
```

Comment:
We can't just `return` here if there's no schema set for the URL. To avoid warnings let's instead check against empty down there where we first try to access the `scheme` key.